src/main/kotlin/com/chattriggers/ctjs/api/triggers/Trigger.kt

```diff
@@ -9,6 +9,8 @@ abstract class Trigger protected constructor(
 ) : Comparable<Trigger> {
     private var priority: Priority = Priority.NORMAL
 
+    private var registered = false
```

Comment:
```suggestion
    var registered = false
        private set
```
Then you can remove the dedicated setter method

Reply:
Uhh this didn't do what I thought it would. I thought Kotlin used an `is` prefix instead of `get` for boolean members? I would like the Java method to still be called `isRegistered`. Maybe if the variable is prefixed with `is` it'll spit it out as is.